app/graphql/types/date_time_type.rb

```diff
@@ -2,5 +2,8 @@
   name 'DateTime'
 
   coerce_input ->(value, _ctx) { Time.zone.parse(value) }
-  coerce_result ->(value, _ctx) { value.utc.iso8601 }
+  coerce_result ->(value, _ctx) { 
+    return value.iso8601 if value.instance_of?(Date)
```

Comment:
It seems like the value stored in the database was of type `Date` which lacks timezone information, so calling `.utc` on the `value` would crash. This just checks the object type.